core/templates/dev/head/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -83,7 +85,7 @@ angular.module('oppia').directive('topicEditorTab', [
 
           $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
 
-          var saveTopicThumbnail = function(imageURI) {
+          var saveTopicThumbnailUrl = function(imageURI) {
```

Comment:
Should this be saveTopicThumbnailImageData? Seems like the thing that's being saved is an "image data" thing.

Also is it an image URI or an image data URI? The former suggests it's a link to a resource, but if that's the case it's a bit weird to call convertImageDataToImageFile on it. Can you clarify what exactly the imageURI represents?

Reply:
Hi @seanlip, it is an image data URI. I'll rename it to saveTopicThumbnailImageData. Thanks.